app/files.go

```diff
@@ -68,6 +72,17 @@ func main() {
         	fmt.Println(err.Error())
         	os.Exit(1)
     }
+
+    flag.Parse()
+    if *cpuprofile != "" {
+        f, err := os.Create(*cpuprofile)
+        if err != nil {
+            fmt.Println(err.Error())
+        	os.Exit(1)
```

Comment:
on CPU profiling you exit if you fail to create the file but for mem profiling the engine does not exit. Perhaps we should be consistent?